client/src/pages/SubmissionsPage.vue

```diff
@@ -97,22 +97,58 @@
               <q-btn data-cy="submission_actions">
                 <q-icon name="more_vert" />
                 <q-menu anchor="bottom right" self="top right">
-                  <q-item clickable>
+                  <q-item
+                    clickable
+                    :disable="cannotAccessSubmission(submission)"
+                    data-cy="review"
+                  >
                     <q-item-section
-                      ><a :href="'submission/review/' + submission.id">{{
-                        $t("submissions.action.review")
-                      }}</a></q-item-section
-                    >
+                      ><a
+                        data-cy="review_link"
+                        :href="'submission/review/' + submission.id"
+                        >{{ $t("submissions.action.review.name") }}</a
+                      >
+                      <q-tooltip
+                        v-if="cannotAccessSubmission(submission)"
+                        anchor="top middle"
+                        self="bottom middle"
+                        :offset="[10, 10]"
+                        class="text-body1"
+                        data-cy="cannot_access_submission_tooltip"
+                      >
+                        {{ $t("submissions.action.review.no_access") }}
+                      </q-tooltip>
+                    </q-item-section>
                   </q-item>
-                  <q-item data-cy="change_status" clickable>
-                    <q-item-section>{{
-                      $t("submissions.action.change_status")
-                    }}</q-item-section>
+                  <q-item
+                    data-cy="change_status"
+                    clickable
+                    :disable="submission.status == 'REJECTED'"
+                  >
+                    <q-item-section data-cy="change_status_item_section"
+                      >{{ $t("submissions.action.change_status.name") }}
+                      <q-tooltip
+                        v-if="submission.status == 'REJECTED'"
+                        anchor="top middle"
+                        self="bottom middle"
+                        :offset="[10, 10]"
+                        class="text-body1"
+                        data-cy="cannot_change_submission_status_tooltip"
+                      >
+                        {{ $t("submissions.action.change_status.no_access") }}
+                      </q-tooltip>
+                    </q-item-section>
```

Comment:
When logged in as a Reviewer, the tooltip does not appear when hovering "change status", and options are still clickable

![image](https://user-images.githubusercontent.com/108834493/187290212-e1b513b2-5499-42c4-a701-2221274be884.png)

Reply:
We decided as a team that we'll do role-based disabling on this part later. For simplicity, we're only disabling this for rejected submissions in the meantime. Reviewers will still have their attempts to change status met with a rejection message. 

<img width="1059" alt="You are not authorized to change the status of this submission. Please contact your administrator." src="https://user-images.githubusercontent.com/2668987/187297386-6f498170-d55f-46b0-9951-661ccc5b0534.png">